### PR TITLE
Container proposal to use themeon on all platforms

### PR DESCRIPTION
**Pull request by abenevaut**
Follow the updated `readme.md` to install and use the Docker container dedicated to themeon.
Actually no image was pushed on Docker hub, but that could be a next step.

### CODE REVIEW

**Comment by pitpit** [comment]
👍

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pitpit) [inline .idea/misc.xml]: I think you should remove this PHPStorm specific directory (`.idea`)

File: README.md [hunk 2]
```diff
@@ -9,6 +9,30 @@ cd themeon
 sudo make install
 ```
 
+## Install Docker container
```
Comment (pitpit) [inline README.md]: I think `Using Docker` would be better as title than`Install Docker container`

File: README.md [hunk 3]
```diff
@@ -9,6 +9,30 @@ cd themeon
...
+
+```shell
+make container
+```
+
+Create an alias to use `themeon.sh` file, to your `.bash_profile` add
```
Comment (pitpit) [inline README.md]: what about simply symlink themeon.sh into local bin. Easier to explain, easier to install. Furthermore, .bash_profile does not exist on every OS/Distrib.

    ln -s /PATH/TO/REPOSITORY/themeon.sh /usr/local/bin/themeon

File: .idea/misc.xml (hunk 1 above)
Comment (42antoine) [reply .idea/misc.xml]: I only push IDE configuration files that allow PHPStorm users to use the exact same settings (enabled and configured tools / plugins). No personal data have been sent, and are ignored globally on my host with the recommended PHPStorm ignore file https://www.gitignore.io/api/jetbrains.

See also https://stackoverflow.com/questions/11968531/what-to-gitignore-from-the-idea-folder

**Comment by abenevaut** [comment]
No maintainer here..

**Pull request closed without merging**